### ISSUE TITLE
Text cut off on Garmin Instinct 2 (and probably 1)

### ISSUE THREAD

**Issue by brzdek** [samuelmr/garmin-abouttime#20]
Hi!
Any chance the way the letters appear on Instinct watches can be changed? Right now the Instinct "circle" cuts off the end of first line a bit.

Not sure which is easier - a specific setup for Instinct or just a toggle in the settings to swap the time with the date? 
I think pushing the text down (below the "circle") and putting the date at the top so it does not look bottom-heavy would be best. Or maybe just pushing the first line to the left would be enough? 

Thanks for considering this! In fact this would be relevant for all the languages directly. 

![PXL_20220906_103134328](https://user-images.githubusercontent.com/112942434/188613654-d5660764-7a8f-4af0-ba5a-586b134e785a.jpg)

**Comment by samuelmr** [comment]
Thanks for the suggestion!

Actually, I just noticed the issue a while ago myself when testing the watch face in descentg1 simulator.

Yesterday, also another user reported the issue about Instinct 2 and suggested just left-aligning all texts.

I'll investigate and test possible solutions when I find the time.

**Comment by brzdek** [comment]
Great to hear!
I have second thoughts about left-aligning, it might not be enough. But I
will happily test and let you know :)

By the way I noticed the mention of Github only after sending the email,
but I already created a new issue there.

Also I have to say - cheers for the great design and the idea to make it
translated. I love the simplicity of this watchface... how it puts you in a
completely different mindset with just a glance is just stunning.

Regards,
Michał

wt., 6 wrz 2022, 21:06 użytkownik Samuel Rinnetmäki <
***@***.***> napisał:

**Comment by samuelmr** [comment]
https://github.com/samuelmr/garmin-abouttime/commit/5a6656691bc4585ed76cd06cc7ad701dd9727e5b sets text alignment to left for all devices with subscreen. It's not very pretty but it was pretty straightforward to implement.

I'll update the Polish and German translations in the Connect IQ store.

**Comment by brzdek** [comment]
Great job, thank you, that looks perfect in most cases! 

I have to ask - is there a possibility to break the first line when it is still too long (so make it 4 lines) or is it not doable easily due to how the app is structured? 

Cheers anyway! 

![PXL_20220911_131821430](https://user-images.githubusercontent.com/112942434/189529833-a7c371df-12fe-4fc1-81f2-eaea8adcfb0a.jpg)

**Comment by samuelmr** [comment]
Thanks for testing and giving feedback.

Adding the fourth line would be cumbersome, but commit https://github.com/samuelmr/garmin-abouttime/commit/afccec81ccfe1738311903ac1c5533545db2f41a drops all text by 20 pixels (and also moves icons left). Texts are centered again. I published the new version 7.1.1 of the Polish translation.

Those hard-coded values are not a very elegant solution to the problem and may cause issues if Garmin releases other devices with subscreens, but let's worry about that later...

**Comment by samuelmr** [comment]
Considering this fixed. Please reopen if needed.
